src/main/resources/application.properties

```diff
@@ -1,2 +1,4 @@
 spring.config.location=classpath:/
 @environment-properties@
+
+server.port=${PORT:8080}
```

Comment:
Should this be included in the changes?

Reply:
This line saves the user a little time and research when port forwarding. With this line, user can run 
"PORT=XXXX mvn spring:boot-run" to port forward